doc/source/structures/vessels/engine.rst

```diff
@@ -357,5 +405,47 @@ Some of the Parts returned by :ref:`LIST PARTS <list command>` will be of type E
     :type: :struct:`Gimbal`
 
     Returns the :struct:`Gimbal` attached to this engine. Only accessible if the gimbal is present (Use :attr:`Engine:HASGIMBAL` to check if available).
+    
+.. attribute:: Engine:ULLAGE
+
+    :access: Get only
+    :type: :ref:`Boolean <boolean>`
+    
+    If RealFuels is installed, returns true if this engine needs ullage, otherwise returns false.
```

Comment:
I could see a user finding the terminology ambiguous here.  What does it return when the engine is a type that requires ullage, but currently DOES have sufficient ullage to ignite properly therefore it doesn't "need" it (it already has it).  i.e. is this a (static) property of the engine type, or is this a current, dynamic property of the engine's current condition.

Reply:
Hmm yes, I se what you mean.
Ullage is a static property if the engine, namely that it is a type that requires ullage.
The related suffix `FUELSTABILITY` is used to check the current condition of the engine.
I'm not at my computer right now, but I will revise the documentation tomorrow to clarify.

Reply:
Thanks.  So far other than the few comments I think I'm ready to merge this, but I'll wait for tomorrow for those changes you mention.